pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -554,3 +547,37 @@ type ClientSpec struct {
 type CleanupPolicySpec struct {
 	DeleteDataDirOnHosts string `json:"deleteDataDirOnHosts"`
 }
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type CephRBDMirror struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata"`
+	Spec              RBDMirroringSpec `json:"spec"`
+	Status            *Status          `json:"status"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type CephRBDMirrorList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata"`
+	Items           []CephRBDMirror `json:"items"`
+}
+
+type RBDMirroringSpec struct {
+	Count int `json:"count"`
```

Comment:
nit: missing comment for consistency

Reply:
Well this file is missing most the comment but ok :)

Reply:
I normally don't comment about missing comments, but the missing comment on `Count` just stood out for some reason :)